src/nes/mapper/mapper033.ts

```diff
@@ -0,0 +1,52 @@
+import {Mapper, MapperOptions} from './mapper'
+import {MirrorMode} from '../ppu/types'
+
+const kMirrorTable = [
+  MirrorMode.VERT,
+  MirrorMode.HORZ,
+]
+
+
+export class Mapper033 extends Mapper {
+	public static create(options: MapperOptions): Mapper {
+		return new Mapper033(options)
+	}
+	
+	constructor(options: MapperOptions) {
+		super(options, 2000)
```

Comment:
This mapper doesn't have SRAM,
so please eliminate second argument: `super(options)`

Reply:
OK, remove under_line and SRAM(2000)